spring-cloud-skipper-server/src/main/resources/application.yml

```diff
@@ -7,6 +7,9 @@ server:
 spring:
   main:
     banner-mode: "off"
+  data:
+    rest:
+      base-path: /api
```

Comment:
shoudl we make this `/api/repositories` ?

Reply:
This is just the base-path, `repositories` will indeed be available at `/api/repositories`